README.md

```diff
@@ -1050,6 +1051,15 @@ The examples of decoupled models shows how to develop and serve
 [decoupled models](../../README.md#decoupled-mode) in Triton using Python backend.
 You can find the complete example instructions in [examples/decoupled](examples/decoupled/README.md).
 
+## AddSub in JAX
```

Comment:
Move this immediately after AddSub numpy/pytorch examples

Reply:
Yes. Updated.